config/environments/deployed.rb

```diff
@@ -26,26 +24,23 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # Compress CSS using a preprocessor.
```

Comment:
Do we not need to define the js_compressor? Or does it default to something we like?

Reply:
Good question.  The absence of a js_compressor setting is the default for new Rails 6 apps and seems to be unnecessary, so that's why I removed it.  I think it's okay because we're handling js compilation via yarn.  I should test deployment though, to ensure that this won't be a problem.  I'll follow up with another comment after that test!

Reply:
So interestingly enough, the latest rails docs say:
```
Possible options for JavaScript compression are :closure, :uglifier and :yui. These require the use of the closure-compiler, uglifier or yui-compressor gems, respectively.
```
I'm not sure what the default is if you don't specify it, but it might actually be a nil default, which would make sense because out Gemfile.lock doesn't include the closure-compiler, uglifier or yui-compressor gems anywhere.

Reply:
👍